### PR TITLE
Feed updates

### PR DESCRIPTION
**Pull request by jbrooksuk**
Updates roumen/feed, also makes sure that we're serving as XML.

### CODE REVIEW

File: app/Http/Controllers/FeedController.php [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
-use Roumen\Feed\Facades\Feed;
+use Roumen\Feed\Feed;
 
 class FeedController extends Controller
 {
     /**
      * Feed facade.
      *
-     * @var \Roumen\Feed\Facades\Feed
+     * @var Roumen\Feed\Feed
```
Comment (GrahamCampbell) [inline app/Http/Controllers/FeedController.php]: cs

File: app/Http/Controllers/FeedController.php [hunk 2]
```diff
@@ -33,12 +33,13 @@ class FeedController extends Controller
      *
      * @return void
      */
-    public function __construct()
+    public function __construct(Feed $feed)
     {
-        $this->feed = Feed::make();
+        $this->feed = app('feed');
```
Comment (GrahamCampbell) [inline app/Http/Controllers/FeedController.php]: ?

File: app/Http/Controllers/FeedController.php [hunk 3]
```diff
@@ -33,12 +33,13 @@ class FeedController extends Controller
...
         $this->feed->title = Config::get('setting.app_name');
```
Comment (GrahamCampbell) [inline app/Http/Controllers/FeedController.php]: hax going on here too

File: app/Http/Controllers/FeedController.php (hunk 1 above)
Comment (jbrooksuk) [inline app/Http/Controllers/FeedController.php]: Gone.

File: app/Http/Controllers/FeedController.php (hunk 3 above)
Comment (jbrooksuk) [inline app/Http/Controllers/FeedController.php]: Typehinting doesn't work lol.

File: app/Http/Controllers/FeedController.php (hunk 2 above)
Comment (jbrooksuk) [inline app/Http/Controllers/FeedController.php]: Honestly, it didn't work.

**Comment by GrahamCampbell** [comment]
@jbrooksuk You never cleared your composer cache.

File: composer.lock [hunk 4]
```diff
@@ -2021,16 +2021,16 @@
         },
         {
             "name": "mccool/laravel-auto-presenter",
-            "version": "4.2.1",
+            "version": "v4.2.1",
```
Comment (GrahamCampbell) [inline composer.lock]: stale composer cache

Comment (jbrooksuk) [inline composer.lock]: I'll redo it.

**Pull request merged**